CHANGELOG.rst

```diff
@@ -3,6 +3,10 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+[0.8.4] in progress
+----------------------
+- Fix ``config maxpeers``
```

Comment:
```suggestion
- Fix ``config maxpeers`` to disconnect peers if limited is lower than before
```
I'd like to suggest a more descriptive entry to people know straight away what was fixed without having to go through the actual PR